operators/azure/cloud_ops_registration.cc

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "ocos.h"
+#include "azure_invokers.hpp"
+#include "azure_triton_invoker.hpp"
+#include "openai_invokers.hpp"
+
+using namespace ort_extensions;
+
+const std::vector<const OrtCustomOp*>& AzureInvokerLoader() {
```

Comment:
nit: I suppose this can be called CloudInvokerLoader since not all custom ops are Azure.

Reply:
wasn't clear how best to name this given they're all using 'AzureExecutionProvider' regardless of the endpoint they're calling. because of that I left Azure in the name, but happy to change it

Reply:
Ok, I see. Let's leave it as-is then. It would've been nice to name the EP as CloudExectuionProvider but that it was voted down for marketing reasons.